### PR TITLE
[INT-205] Do in channel check properly

### PR DESCRIPTION
**Pull request by archessmn**
…heck
^ This was left over from the weird commit message wrapping but sums it up pretty well lmao

### CODE REVIEW

File: app/(authenticated)/calendar/new/actions.ts [hunk 1]
```diff
@@ -100,20 +101,33 @@ export const createEvent = wrapServerAction(
...
+          // });
+
+          // if (channel_members.ok) {
+          //   if (!channel_members.members?.includes(slackUser.provider_key)) {
+          //     await slackApp.client.conversations.invite({
+          //       channel: slack_channel_id,
+          //       users: slackUser.provider_key,
+          //     });
+          //   }
+          // }
```
Comment (markspolakovs) [inline app/(authenticated)/calendar/new/actions.ts:130]: Can this go?

File: lib/slack/index.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { z } from "zod";
+
+export function parseAsSlackError(error: unknown) {
```
Comment (markspolakovs) [inline lib/slack/index.ts:3]: Clever, though doesn't `@slack/bolt`/`@slack/web-api` have a `CodedError`/`isCodedError` for this?

File: app/(authenticated)/calendar/new/actions.ts (hunk 1 above)
Comment (archessmn) [reply app/(authenticated)/calendar/new/actions.ts:130]: Oops (yes)

**Review by markspolakovs — APPROVED** [review]
Okay

**Comment by markspolakovs** [comment]
Though I'd rather not have the as statements

**Comment by archessmn** [comment]
Tough

**Pull request merged**